app/src/main/java/org/ea/sqrl/activites/identity/ImportActivity.java

```diff
@@ -115,8 +115,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
                     if(newIdentityId != 0) {
                         if(firstIdentity) {
-                            mDbHelper.updateIdentityName(ImportActivity.this, newIdentityId,
```

Comment:
Why did we remove this? Shouldn't we set the first identity name to "My identity"?

Reply:
Since the previous PR, this happens already in `newIdentity()`, so it is already taken care of at that point.